.github/workflows/scripts/biome_disable_check.py

```
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)